truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/nodes/LoopNode.java

```diff
@@ -107,6 +110,8 @@
  * @since 0.8 or earlier
  */
 public abstract class LoopNode extends Node {
```

Comment:
Right, `TVMCI.initializeCallTarget` isn't perfect. On the other hand, I don't want to introduce another TVMCI class right now. Here is possible solution: https://github.com/graalvm/truffle/pull/139/commits/f73c774218806694b54b8ad7925a53c2ce7de121 - e.g. let's rename the method to `installRootNode` and then it starts to be more appropriate. The method works on RootNode and thus the method is now in the right (e.g. `com.oracle.truffle.api.nodes`) package.